scripts/lib/mkcloud-common.sh

```diff
@@ -539,3 +539,42 @@ function dist_to_image_name
 }
 
 # ---- END: functions related to repos and distribution settings
+
+function qacrowbarsetup_help
```

Comment:
@aspiers 
Agreed, the message should be more specific.

Yes, the name `mkcloud-common.sh` is not the best. This file contains functions that are common to `mkcloud` and `qa_crowbarsetup.sh`. Maybe `mkc-qac-common.sh` or `m-q-common.sh` would be better. But having a separate file like `qacrowbarsetup-shared.sh` is also a nice idea. It's yet another file, but would also make sense.